src/mobile_crawler/infrastructure/ai_interaction_service.py

```
@@ -318,8 +327,11 @@ def _parse_ai_response(self, response_text: str) -> AIResponse:
         if not response_text:
             raise ValueError("Empty response from AI model")
         
+        # Strip <think>...</think> tags which are output by some local models
+        import re
```

Ruff (0.15.2)

[error] Redefinition of unused `re` from line 4: `re` redefined here

Remove definition: `re`

(F811)